scripts/stupid_roomba.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+import rospy
+import tf2_ros as tf2
+
+from geometry_msgs.msg import TransformStamped
+from iarc7_msgs.msg import OdometryArray
+from nav_msgs.msg import Odometry
+
+if __name__ == '__main__':
+    rospy.init_node('stupid_roomba')
```

Comment:
considering our code is public do we really want to start the convention of using childish names for things?